cobc/tree.h

```diff
@@ -914,6 +914,7 @@ struct cb_field {
 	int			size;		/* Field size */
 	int			level;		/* Level number */
 	int			memory_size;	/* Memory size */
+	int			compx_size;	/* Original COMP-X byte size */
```

Comment:
Couldn't we just use f->pic->size for this rarely used definition, instead of adding an extra field to all variables?